.travis.yml

```diff
@@ -55,12 +55,12 @@ matrix:
     - (cd diesel_migrations && cargo rustc --no-default-features --features "lint dotenv sqlite postgres mysql" -- -Zno-trans)
     - (cd diesel_infer_schema && cargo rustc --no-default-features --features "lint dotenv sqlite postgres mysql" -- -Zno-trans)
     - (cd diesel_compile_tests && travis-cargo test)
-  - rust: nightly-2017-09-20
+  - rust: nightly-2017-11-07
```

Comment:
Why not today?

Reply:
I bumped because crates.io did and this was the date they were using and `rustup update nightly` failed.